support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -433,7 +433,7 @@ export const productCatalogDescription: Record<
 	SubscriptionCard: {
 		label: 'Collect in store with a subscription card',
 		thankyouMessage: paperThankyouMessage,
-		landingPagePath: '/subscribe/paper#Collection',
+		landingPagePath: '/subscribe/paper',
```

Comment:
Was this removed because collection is now the default? Should the `#HomeDelivery` hash param be added in the `HomeDelivery` section below?

Reply:
Think we agreed to leave this as just the path (no hash tag ) for now